### ISSUE TITLE
The fake nuke disk has a SLIGHTLY different sprite than the real nuke disk (at least on kilo)

### ISSUE THREAD

**Issue by nikothedude** [tgstation/tgstation#58140]
## Round ID: 
[159552](https://scrubby.melonmesa.com/round/159552)

## Testmerges: 
#58096: '[15.ai] [April Fools] The Nanotrasen Voice Module subscription has expired.' by Iamgoofball at commit 7a192b7550

## Reproduction: 
Step 1: Be a traitor (or be captain on kilo and get a free traitor item for some reason because the fake disk spawns in caps office from my understanding)
Step 2: Hatch a genius plan to give the fake nuke disk to sec in order to secure their trust and maybe even get some gamer gear
Step 3: Show the disk to a sec officer, who looks at it, looks back up at you, immediately batons you into stamcrit and beats you to death because the normal disk is slightly lopsided to the left, but the fake disk is lopsided to the right.
Step 4: Ahelp them for killing you for no raisin
Step 5: Cry as you're told it's an IC issue
(None of this happened, this is all a hypothetical)

Basically, the nuke disk is slightly lopsided to the left. The fake disk is lopsided to the right, meaning it's obviously fake. It's not much; I had to look at it pixel-by-pixel to see if it was actually different. 

Here's an image: Real on left, fake on right. Notice how the real disk has an extra line of blue pixels to the left along just being bigger, and the fake disk has an extra line of green pixels to the right.
https://i.imgur.com/hwW8qeX.png

Bigger image, showing the pixels: https://i.imgur.com/vZEnsUb.png

At any rate, this is a CRITICAL bug that will, without a doubt, result in the complete destruction of TG if not fixed immediately.

**Comment by nikothedude** [comment]
Fixed the image links, cause they broke

**Comment by nikothedude** [comment]
I've been told this might be stretch to fit fucking it up, and it miiight be

**Comment by nikothedude** [comment]
Yep I'm fucking stupid, WOOOOOOOOOOOOOOOOO
